### ISSUE TITLE
Fika Support

### ISSUE THREAD

**Issue by christopher-talke** [ekky-llc/spt-raid-review#4]
Need to see what work needs to be done to support Fika.

**Comment by olli991** [comment]
You could join the Discord of Fika: https://discord.gg/project-fika 
There is a dedicated Section for mods and their developers. It's rly active and you'll get direct feedback and probably also suppord from other developers. 
Could be an "easy" way to dive into this.

**Comment by christopher-talke** [comment]
> You could join the Discord of Fika: https://discord.gg/project-fika 
> There is a dedicated Section for mods and their developers. It's rly active and you'll get direct feedback and probably also suppord from other developers. 
> Could be an "easy" way to dive into this.

Yeah - I'm in there, I don't have anyone to directly test Fika with atm, however, it's something I'm planning to setup for myself over the weekend to properly test.

**Comment by olli991** [comment]
We (a friend an me) can test stuff with fika. Also just setup the stuff on an VPS, so it's completly external :). U can find me with the same user name on discord as well

**Comment by olli991** [comment]
Read about the current update and first step in support for fika.

Just a thinking of mine for further options:
-shouldn't it be possible to detect the current raid host and only accept data from host if there are multiply hosts connected? 

A simple switch in the F12 menu for enabled or disabled functionality could be an fast solution but manually for the time being.
